admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -2,83 +2,165 @@
+::
+
+  sudo service apache2 stop
+
+Download the Latest Installation
+--------------------------------
+
+Download the latest ownCloud server release from `owncloud.org/install/`_ into an empty directory **outside** of your current installation.
     
-.. note:: Enterprise users must download their new ownCloud archives from 
-   their accounts on `<https://customer.owncloud.com/owncloud/>`_
-   
-3. Stop your Web server.
+.. note:: 
+   Enterprise users must download their new ownCloud archives from their accounts on `<https://customer.owncloud.com/owncloud/>`_.
 
-4. Rename your current ownCloud directory, for example ``owncloud-old``.
+Setup the New Installation
+--------------------------
 
-5. Unpacking the new archive creates a new ``owncloud`` directory populated 
-   with your new server files. Copy this directory and its contents to the 
-   original location of your old server, for example ``/var/www/``, so that 
-   once again you have ``/var/www/owncloud``.
+Not all installations are the same, so we encourage you to take one of two paths to upgrade your ownCloud installation. 
+These are the standard upgrade and the power user upgrade.
 
-6. Copy the ``config.php`` file from your old ownCloud directory to your new 
-   ownCloud directory.
+If you're reasonably new to ownCloud, or not too familiar with upgrading an ownCloud installation, please follow the standard upgrade.
+Otherwise, take the approach that you're most comfortable with, likely the power
+user upgrade.
 
-7. If you keep your ``data/`` directory in your ``owncloud/`` directory, copy 
-   it from your old version of ownCloud to your new ``owncloud/``. If you keep 
-   it outside of ``owncloud/`` then you don't have to do anything with it, 
-   because its location is configured in your original ``config.php``, and 
-   none of the upgrade steps touch it.
+.. note::
+   Regardless of which approach that you take, they will both assume that your existing ownCloud installation is located in the default location: ``/var/www/owncloud``.
 
-8. If you are using 3rd party applications, look in your new ``owncloud/apps/`` 
-   directory to see if they are there. If not, copy them from your old ``apps/``
-   directory to your new one. Make sure the directory permissions of your third
-   party application directories are the same as for the other ones.
+The Standard Upgrade
+~~~~~~~~~~~~~~~~~~~~
 
-9. Restart your Web server.
+Delete all files and folders in your existing ownCloud directory (``/var/www/owncloud``) — **except** ``data`` and ``config``. 
 
-10. Now launch the upgrade from the command  line using ``occ``, like this 
-    example on CentOS Linux::
-    
-     sudo -u apache php occ upgrade
+.. attention:: Don't keep the ``apps`` directory.
+
+With those files deleted, extract the archive of the latest ownCloud server, over the top of your existing installation.
+
+::
+
+  # Extract the .tar.bz2 archive
+  tar -jxf owncloud-10.0.3.tar.bz2 -C /var/www/ --exclude config
```

Comment:
You don't need to exclude the config dir here and below. It only contains the config.sample.php which could contain newer sample config options.

Reply:
Fair point.